_posts/2020-11-02-Facialrecognition-counterterrorism.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Documents Reveal 'Counter-Terrorism' Funds Flowed to Clearview AI"
+image: images/biometrics/project_header.jpg
+author: lucyparsonslabs
+teaser: "Documents obtained by LPL appear to confirm DHS is investigating journalists for their reporting"
+tags:
+  - BIPA
+  - privacy
+  - biometrics
+---
+
+Documents obtained exclusively by Lucy Parsons Labs reveal Chicago-area police
+entered into a $50,000 dollar contact with a controversial facial recognition
+company, Clearview AI, using counter-terrorism funds. Chicago Police obtained
+the [funding](https://beta.documentcloud.org/documents/20400239-cpic_clearview_ai_refund_check_and_cancellation_memo) under the Urban Areas Security Initiative (UASI) which the
+Department of Homeland Security’s (DHS) funnels to local police in order to
+“prevent, protect against, mitigate, respond to, and recover from acts of
+terrorism.” Clearview’s contract with the Chicago Police Department’s (CPD) was
+[cancelled](https://chicago.suntimes.com/crime/2020/1/29/21080729/clearview-ai-facial-recognition-chicago-police-cpd) after public outcry over its [controversial data](https://beta.documentcloud.org/documents/20400239-cpic_clearview_ai_refund_check_and_cancellation_memo) collection in
+violation of the Illinois’ biometrics privacy law.
+
+UASI funding to Clearview was approved by the First Deputy Superintendent of the
+Chicago Police Department, Anthony Riccio, the second highest [ranking official](https://twitter.com/FOX2News/status/1323111377725198336)
+within CPD. Riccio approved the funding for the software for use in the Crime
+Prevention and Information Center (CPIC). CPIC is the Chicagoland-area “fusion
+center created after 9/11 to “improve the sharing of intelligence information
+related to terrorism among federal, state and local authorities.” Over seventy
+centers exist across the country and internationally. Fusion centers have long
+been criticized for their documented history of [spying](https://theintercept.com/2020/08/17/blueleaks-california-ncric-black-lives-matter-protesters/) on First Amendment
+activity and [fabricating](https://theintercept.com/2020/08/24/fbi-fusion-center-environmental-wind/) [threats](https://theintercept.com/2020/06/26/blueleaks-minneapolis-police-protest-fears/) against police and [corporations](https://theintercept.com/2020/10/16/wildfires-forest-jihad-blueleaks/).
+
+Clearview AI is facing a number of [legal challenges](https://theintercept.com/2020/10/16/wildfires-forest-jihad-blueleaks/) after its indiscriminate
+collection of [biometric data](https://www.nytimes.com/2020/01/18/technology/clearview-privacy-facial-recognition.html) was revealed on the front page of the New York
+Times. Among the legal arguments raised by Clearview include that the company is
```

Comment:
s/include/is

Reply:
Maybe reword this sentence to something like "Clearview claims that..." Can do this after merge if needed.